code/modules/mob/living/death.dm

```diff
@@ -1,93 +1,26 @@
-/mob/living/gib(no_brain, no_organs, no_bodyparts)
-	var/prev_lying = lying
-	if(stat != DEAD)
-		death(TRUE)
-
-	if(!prev_lying)
-		gib_animation()
-
-	spill_organs(no_brain, no_organs, no_bodyparts)
-
-	if(!no_bodyparts)
-		spread_bodyparts(no_brain, no_organs)
-
-	spawn_gibs(no_bodyparts)
-	qdel(src)
+/mob/living/gib

```

Comment:
ah, got the compile error. you forgot to add arguments to this proc you silly goose